src/apispec/ext/marshmallow/field_converter.py

```diff
@@ -289,19 +289,12 @@ def field2range(self, field, **kwargs):
             )
         ]
 
-        attributes = {}
-        for validator in validators:
-            if validator.min is not None:
-                if hasattr(attributes, "minimum"):
-                    attributes["minimum"] = max(attributes["minimum"], validator.min)
-                else:
-                    attributes["minimum"] = validator.min
-            if validator.max is not None:
-                if hasattr(attributes, "maximum"):
-                    attributes["maximum"] = min(attributes["maximum"], validator.max)
-                else:
-                    attributes["maximum"] = validator.max
-        return attributes
+        min_attr, max_attr = (
+            ("minimum", "maximum")
+            if marshmallow.fields.Number in type(field).mro()
```

Comment:
This is not perfectly generic as it assumes only subclasses of `Number` field  may represent numbers.